qcelemental/models/results.py

```diff
@@ -270,6 +281,14 @@ class AtomicResultProtocols(ProtoModel):
         WavefunctionProtocolEnum.none, description=str(WavefunctionProtocolEnum.__doc__)
     )
     stdout: bool = Field(True, description="Primary output file to keep from a Result computation")
+    error_correction_policy: ErrorCorrectionEnum = Field(
+        ErrorCorrectionEnum.all, description="Policy used to determine which error correction routines to use"
+    )
+    error_correction_settings: Optional[Dict[str, bool]] = Field(
```

Comment:
This looks like a whitelist/blacklist and a True/False per setting which would be similar to the white/blacklisting? Perhaps `ErrorCorrectionEnum.explicit` for this or am I way off base.

Reply:
Yea, this just occurred to me while I as writing a convoluted description of the changes I made. I'm working on a simplified version

Reply:
Maybe I'm planning UI too far in advance, but for the enum, I was thinking none/all/safe/.../explicit so we can group common ones like "orbitals_and_eigenvalues" does for wfn.

Reply:
Would the idea be that we can label specific error corrections with tags such as "safe"?

Reply:
I wasn't thinking labels on each correction as just shortcuts to groups https://github.com/MolSSI/QCElemental/blob/master/qcelemental/models/results.py#L395-L396. But the dict is the more important at this stage

Reply:
I see. That sort of a single level identifier mapping to more than one type of error could be a good model and would be the most beneficial if we have many engines that throw the same type of error. 

I was picturing attaching some kind of "danger level" to certain error corrections, depending on how likely they are to affect the accuracy of the output. That would be more beneficial if we have different error correction routines each code. 

I'm not sure which of these pictures (if accurate) is more accurate. Should we punt on this until we have more error classes to go off of?

Reply:
I think both the pictures you describe will likely show up (the symm cmd that started all this is nwc only but use-more-opt-cycles will be valid for many progs). I'm just advocating that the non-dict field not be a bool but instead be an enum (none/all/explicit for starters). But if you and @dgasmith are happy with bool for now, I won't insist.